### ISSUE TITLE
Error in running CARD_SCMapping

### ISSUE THREAD

**Issue by michaeljpitcher** [YingMa1993/CARD#5]
Hi,

I've been following the tutorial and run into an error when running CARD_SCMapping():

```
Error in SummarizedExperiment(...) : 
  the rownames and colnames of the supplied assay(s) must be NULL or identical to those of the
  SummarizedExperiment object (or derivative) to construct

```
I think this may be occurring when the SingleCellExperiment is created in CARD_SCMapping, due to a mismatch in the colnames of count_CT (which have spot and coordinates appended to them) and the rownames of the colData dataframe (which do not have spot and coordinates appended).

**Comment by YingMa0107** [comment]
Hi @michaeljpitcher, 

Thank you very much for your interest in our package! I have previously run the tutorial successfully on both MacOS and Linux systems (Ubuntu) without errors. I could not reproduce your error even I tried different versions of SingleCellExperiment. But your suggestion is great, thanks for this! I've already added one line to add the rownames of colData frame. Let me know if this solves the issue!
